### ISSUE TITLE
[Android] Play link: is invalid!

### ISSUE THREAD

**Issue by Sadjiwo** [WhatsApp/stickers#247]
I had read readme.md with carefully and following the instructions.
I can build the app and install it on my device with marshmellow inside.
Firstly, all of my stickers show up with successfully. No any problem yet.
But, when I try to tab the "Add to whatsapp" button, I got error message:
"There's a problem with this sticker pack and it can't be added to WhatsApp".
And then I tab the "Ok" button, the other error message show up: "Play link: is invalid!"
How can I fix the problem.
Thanks.

*I'm so sorry with my english.

**Comment by suriya27** [comment]
Its just a minor error Here is a fix :
1) Go to assets folder.
2) Open contents.json
3) "android_play_store_link": " ",
4) In this Line add the play store link as mentioned in Readme.md with your package name.
5) Example :
"android_play_store_link":  "https://play.google.com/store/apps/details?id=com.example.samplestickerapp"

**Comment by suriya27** [comment]
Its just a minor error Here is a fix :
1) Go to assets folder.
2) Open contents.json
3) "android_play_store_link": " ",
4) In this Line add the play store link as mentioned in Readme.md with your package name.
5) Example :
"android_play_store_link":  "https://play.google.com/store/apps/details?id=com.example.samplestickerapp"

**Comment by suriya27** [comment]
Its just a minor error Here is a fix :
1) Go to assets folder.
2) Open contents.json
3) "android_play_store_link": " ",
4) In this Line add the play store link as mentioned in Readme.md with your package name.
5) Example :
"android_play_store_link":  "https://play.google.com/store/apps/details?id=com.example.samplestickerapp"

**Comment by Sadjiwo** [comment]
Hi, @suriya27 
Thanks for great steps.
But I don't want to release my app first to play store. I just want play it for my self only. Can I do that without submit to play store?

**Comment by suriya27** [comment]
Yes bro as I mentioned just add this with your pack name :"https://play.google.com/store/apps/details?id="
No need to Add to Playstore :
com.example.samplestickerapp

Reason : Whatsapp Checks For a Valid Link Not Check On PlayStore It Only Need That Playstore Hyperlink For Some Reasons

**Comment by Sadjiwo** [comment]
Thank you very much, @suriya27
Nice reason.
I will try it just like what your mentioned, add the link in your quote. I hope this my first app can run like what it has todo.
See you soon.

**Comment by RojPro** [comment]
hello 
every think is work and folder 1 add to whatsapp but folder 2,3,4,5 is cant add and show me this Message(There's a problem with this sticker pack and it  can't be added to whatsapp)
how ican fix that ??

![screenshot_20181208-150820_whatsapp](https://user-images.githubusercontent.com/37383703/49691798-06d0ba80-fb53-11e8-9091-1a9c9207f40d.jpg)
![screenshot_20181208-150912_dissticker](https://user-images.githubusercontent.com/37383703/49691799-07695100-fb53-11e8-9b8c-f2d8e924b042.jpg)
